Base/usr/share/man/man1/pkg.md

```diff
@@ -0,0 +1,34 @@
+## Name
+
+pkg - Package Manager
+
+## Synopsis
+
+```**sh
+$ pkg [-l] [-d] [-u] [-v] [-q package]
+```
+
+## Description
+
+This program can list installed packages and query for [available packages](https://github.com/SerenityOS/serenity/blob/master/Ports/AvailablePorts.md). The [Ports for SerenityOS website](https://ports.serenityos.net) has more detailed information about available packages.
```

Comment:
`https://ports.serenityos.net` is an official website of the SerenityOS project. I am not against adding it, but maybe we should wait for @awesomekling to approve on this.